.github/workflows/integration-windows.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - name: Set up Go
```

Comment:
they were originally kept behind the review submission since it was taking too long, but i agree it's a good idea to run them at the beginning of the process